### PR TITLE
Initial release, first try…

### PR DESCRIPTION
**Pull request by lapineige**
Very much work in progress, I have no idea if this is going to work… Expect it won't, and it will probably not even remove itself correctly.

**DO NOT TEST THIS IN PRODUCTION ENVIRONMENT**

### CODE REVIEW

**Comment by lapineige** [comment]
!testme

**Comment by lapineige** [comment]
!testme

**Comment by lapineige** [comment]
!testme

**Comment by lapineige** [comment]
!testme

**Comment by lapineige** [comment]
!testme

**Comment by lapineige** [comment]
!testme

**Comment by lapineige** [comment]
!testme

**Comment by lapineige** [comment]
!testme

**Comment by lapineige** [comment]
@YunoHost-Apps/apps-group main CI is buggy apparently, always giving a "!!! Another analyseCI process is currently using the lock ./CI-3.lock !!!" :(. Do you know what's happening ?

**Comment by lapineige** [comment]
!testme

**Comment by lapineige** [comment]
!testme

**Comment by lapineige** [comment]
Hello @YunoHost-Apps/apps-group I can't find how we are supposed to setup a postgreSQL database now with the package v2 manifest ?
I doesn't find an user for it, it doesn't seem installed correctly… I need help :sweat_smile:

**Comment by lapineige** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+#=================================================
+# CREATE A POSTGRESQL DATABASE
+#=================================================
+ynh_script_progression --message="Creating a PostgreSQL database..." --weight=1
+
+db_name=$(ynh_sanitize_dbid --db_name=$app)
+db_user=$db_name
+ynh_app_setting_set --app=$app --key=db_name --value=$db_name
+ynh_psql_test_if_first_run
+ynh_psql_setup_db --db_user=$db_user --db_name=$db_name
```
Comment (tituspijean) [inline scripts/install:125]: These are not needed in packaging v2. 😇

Comment (lapineige) [reply scripts/install]: Oh I see :D

I still don't understand how v2 works, it seems to ease a lot of things :)

**Review by lapineige — COMMENTED** [review]
Please, don't "clean" useful comments.
I'll add them again.

**Pull request merged**